src/modules/fancyzones/editor/FancyZonesEditor/GridEditor.xaml.cs

```diff
@@ -227,8 +227,8 @@ private void OnGridDimensionsChanged()
             SetupUI();
         }
 
-        private double _dragX = 0;
-        private double _dragY = 0;
+        private double _dragX;
+        private double _dragY;
```

Comment:
This is not acceptable, we should be allowed to set default values for private properties.
In any case, this commit cannot be merged unless the private properties initialization is moved inside the constructor.
Assuming that an un-initialized property is implicitly initialized to 0 or null is also not acceptable because when we do a core review how can we know if the default initialization to 0/null is intentional or not?

Reply:
This is how the code base for anything that is .NET is already designed minus this code base.  tons of other vars through FZ are like this already

Reply:
We need to explicitly initialize variables, otherwise code reviews and code maintainability become problematic and a source of bugs.

Reply:
Also I don't understand what is the reasoning to make illegal to:
 - initialize a variable outside of the constructor
 - not allow to initialize explicitly a variable to zero or null

Reply:
you 100% can init a var outside the constructor.  This rule is saying don't init to default value. 
https://github.com/microsoft/PowerToys/blob/master/src/modules/colorPicker/ColorPickerUI/ViewModels/ColorEditorViewModel.cs#L58 is an example of where it is set to non-default

Why?  it is the c# style.  We could set the rule to be ignored but i don't see why.

Reply:
The reason is very simple: if default initialization is implicit it's impossible to read the code and understand if it was done intentionally or not.

Reply:
This is how the entire solution is done.  There are a lot of things that could be crisper but right now, this is part of the c# style guidelines and the guidelines this project adheres to.  If we want to change it, please lets create an issue and have a discussion.

For now, this is our standard we are adhering to as the entire solution already does.